{"name": "test_mark_function_by_process_tag_working_exception", "status": "broken", "statusDetails": {"message": "Exception: Autogenerated Exception.", "trace": "self = <tests.test_report.TestClass object at 0x7f284c39e4e0>\n\n    @pytest.mark.test_process_marks(\"Working\")\n    def test_mark_function_by_process_tag_working_exception(self):\n>       raise Exception(\"Autogenerated Exception.\")\nE       Exception: Autogenerated Exception.\n\ntests/test_report.py:20: Exception"}, "attachments": [{"name": "Python image", "source": "7fe03332-e11f-4143-acb5-c107812ce5e3-attachment.png", "type": "image/png"}, {"name": "log", "source": "84debf7e-d387-4b10-8c60-0cbb82c5cf87-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "02d4ebd4-36b5-4578-afe0-9906f0704840-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "6e23dfe4-aba7-40b3-86cb-674a5723d683-attachment.txt", "type": "text/plain"}], "start": 1554910516274, "stop": 1554910516274, "uuid": "b66b1f11-85b5-4d4b-bb0f-9981133ebb3b", "historyId": "30ba1d34e3af98868088428607c07b34", "fullName": "tests.test_report.TestClass#test_mark_function_by_process_tag_working_exception", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_report"}, {"name": "subSuite", "value": "TestClass"}, {"name": "host", "value": "travis-job-6469e431-8402-46cb-8c3f-d71eca348f85"}, {"name": "thread", "value": "3636-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_report"}]}